docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       - ${VOLUMES}:/volumes
       - ${DATASETS}:/datasets
       - ${EXPERIMENTER}:/d3m-experimenter
+      - ${D3M}:/d3m
```

Comment:
why is this needed? it is installed in the base image

Reply:
Yeah I think I was tired of adding it into all my branch work and then just decided to leave it.  I'll take it out